docs/rfcs/xxxx-invisible-minimal-sync.md

```diff
@@ -0,0 +1,91 @@
+# Invisible minimal sync
+
+## 0 - Basic considerations
+
+Considering the following :
+
+1. Alice creates a file in location `/foo/big.txt`
+2. Alice write a big amount of data into `/foo/big.txt`
+3. Alice create another file in location `/foo/small.txt`
+4. Bob wants to access to `/foo/small.txt` asap
+
+Here the simplest way to synchronize the data is:
+
+1. Sync `/foo/small.txt`'s file manifest
+2. Sync `/foo/big.txt`'s file manifest
+3. Sync `/foo`'s folder manifest
+
+However this is bad because `/foo`'s changes cannot be accessed by Bob as long as
+`/foo/big.txt` is not done with synchronization.
+
+For that reason Parsec currently has slightly more complexe way of doing synchronization:
+
+1. Sync `/foo/small.txt`'s file manifest
+2. Sync a minimal version `/foo/big.txt` (i.e. file manifest corresponding to an empty file)
+3. Sync `/foo`'s folder manifest
+4. Sync of `/foo/big.txt` actual data as version 2 of it file manifest
+
+## 1 - The issues with minimal sync
+
+The minimal sync approach is great to reduce synchronization congestion, however it creates
+an empty files manifest that cannot be differentiate from an actual legitimate empty file
+(e.g. by doing a `touch foo.txt`).
+
+### 1.1 - Issue in prod
+
+This empty file is especially bad for non-text documents because the software that should
+open them is not expecting the file to be empty, hence bad error message that make Parsec look broken :'(
```

Comment:
```suggestion
This empty file is especially inconvenient for non-text documents because the software that
open them may not be expecting the file to be empty, resulting in an error message that makes Parsec look broken :'(
```